EIPS/eip-4519.md

```diff
@@ -4,7 +4,7 @@ title: Non-Fungible Tokens Tied to Physical Assets (SmartNFT)
 description: Standard interface for non-fungible tokens representing physical assets that can generate or recover their own accounts and obey users.
 author: Javier Arcenegui (@Hardblock-IMSE-CNM), Rosario Arjona <arjona@imse-cnm.csic.es>, Roberto Román <roman@imse-cnm.csic.es>, Iluminada Baturone <lumi@imse-cnm.csic.es>
 discussions-to: https://ethereum-magicians.org/t/new-proposal-of-smart-non-fungible-token/7677
-status: Draft
+status: Review
```

Comment:
I can't make comments on parts that are unchanged, so I'll be putting them here:

- L32-L40: The images do not appear to be correctly loaded, would you mind double-checking the links?
- L56: I believe you have an extra space in `engagedWithOwner`
- Throughout: You are using quotation marks to refer to code/states. It's recommended to use backticks (\`) instead. Here's an example:
  
  L203: ...some functions such as "startUserEngagement" are only...
  L203 (with backticks): ...some functions such as `startUserEngagement` are only...
- Throughout: There are quite a few grammar errors. I would recommend running the paper through a grammar checker and making any appropriate changes.